airqo-platform/AirQo-frontend
# Netmanager device status filter

What does this PR do?
Shows list of online and offline devices - Blue = online, grey = offline
What JIRA task is related to this PR?
https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?assignee=5d91eed256bb950dd6489270&selectedIssue=NET-5

How do I test out this PR?
Git clone this repo cd AirQo-frontend/locate Then run npm install to get all the necessary dependencies Afterwards run npm run local-mac for MAC users and npm run local-pc for windows users

Other considerations?
You should also run the device-monitoring service - map-filter-api branch.
